src/Illuminate/Database/Migrations/Migrator.php

```diff
@@ -73,17 +81,20 @@ class Migrator
      * @param  \Illuminate\Database\ConnectionResolverInterface  $resolver
      * @param  \Illuminate\Filesystem\Filesystem  $files
      * @param  \Illuminate\Contracts\Events\Dispatcher|null  $dispatcher
+     * @param  \Illuminate\Contracts\Container\Container|null  $container
      * @return void
      */
     public function __construct(MigrationRepositoryInterface $repository,
                                 Resolver $resolver,
                                 Filesystem $files,
-                                Dispatcher $dispatcher = null)
+                                Dispatcher $dispatcher = null,
+                                ?Container $container = null)
```

Comment:
Don't use the questionmark.

Reply:
Why?

Reply:
With strict mode we will need to pass `null` values explicitly to build Migrator instance

Reply:
Removed questionmark